lib/IRGen/IRGenSIL.cpp

```diff
@@ -817,6 +818,17 @@ class IRGenSILFunction :
                                     SILType SILTy, const SILDebugScope *DS,
                                     VarDecl *VarDecl, SILDebugVariable VarInfo,
                                     IndirectionKind Indirection = DirectValue) {
+    // TODO: fix demangling for C++ types.
```

Comment:
Is there a separate PR for these changes to the compiler?

Reply:
No, there's not. I'll make one.

Reply:
Here it is: #32815.